src/fans/segments/getSegment.ts

```
@@ -0,0 +1,72 @@
+import { config, configuration } from "config";
```

ESLint

[error] 'config' is defined but never used.

(@typescript-eslint/no-unused-vars)